app/mm-redux/reducers/entities/apps.ts

```diff
@@ -11,8 +11,8 @@ import {validateBindings} from '@utils/apps';
 export function bindings(state: AppBinding[] = [], action: GenericAction): AppBinding[] {
     switch (action.type) {
     case AppsTypes.RECEIVED_APP_BINDINGS: {
-        validateBindings(action.data);
-        return action.data || [];
+        action.data = validateBindings(action.data);
+        return action.data;
```

Comment:
This seems weird that we are editing `action` directly. Some other reducer may access this same action's data at some point right? I don't know if we have control over which reducer receives the action first.

Reply:
validateBindings does update the input value. This is just to be coherent with a similar change in webapp. We can create a new variable here bindings, but the input value would still be updated.